plugins/workspace-minimap/src/minimap.ts

```diff
@@ -27,44 +27,55 @@ const BlockEvents = new Set([
  * your code looks like, and how it is organized.
  */
 export class Minimap {
-    protected primaryWorkspace: Blockly.WorkspaceSvg;
-    protected minimapWorkspace: Blockly.WorkspaceSvg;
+    private primaryWorkspace: Blockly.WorkspaceSvg;
+    minimapWorkspace: Blockly.WorkspaceSvg;
     private onMouseMoveWrapper: Blockly.browserEvents.Data;
     /**
-     * Constructor for a minimap
-     * @param workspace The workspace to mirror
+     * Constructor for a minimap.
+     * @param workspace The workspace to mirror.
      */
     constructor(workspace: Blockly.WorkspaceSvg) {
       this.primaryWorkspace = workspace;
-      const options = {
-        readOnly: true,
-        move: {
-          scrollbars: {
-            vertical: true,
-            horizontal: true,
-          },
-          drag: false,
-          wheel: false,
-        },
-      };
-      this.minimapWorkspace = Blockly.inject('minimapDiv', options);
     }
 
     /**
      * Initialize.
      */
     init(): void {
+      // Create a wrapper div for the minimap injection.
+      const minimapWrapper = document.createElement('div');
+      minimapWrapper.id = 'minimapWrapper_' + this.primaryWorkspace.id;
+      minimapWrapper.className = 'minimapWrapper';
+
+      // Make the wrapper a sibling to the primary injection div.
+      const developerDiv = this.primaryWorkspace.getInjectionDiv().parentNode;
+      developerDiv.appendChild(minimapWrapper);
+
+      // Inject the minimap workspace
+      this.minimapWorkspace = Blockly.inject(minimapWrapper.id,
+          {
+            readOnly: true,
+            move: {
+              scrollbars: true,
+              drag: false,
+              wheel: false,
+            },
+            zoom: {
+              maxScale: null,
+              minScale: null,
+            },
```

Comment:
Can you add some two comments here, one about the move options and one about the zoom options? If you don't have context the reason these are set this way would be confusing!

Reply:
Comments look perfect!